lumigator/python/mzai/sdk/tests/BUILD.pants

```diff
@@ -3,7 +3,14 @@ python_test_utils(name="sdk_test_utils", sources=["conftest.py", "helpers.py"])
 python_tests(
     name="sdk_tests",
     sources=["**/test_*.py"],
+    # sources=["**/test_*.py", "**/int_test_*.py"],
     dependencies=[":data-json", "3rdparty/python:3rdparty#pydantic"],
 )
 
-resources(name="data-json", sources=["data/*.json"])
+python_tests(
```

Comment:
Since I'm looking at this branch for my own local-up issues... Do we need this pants file at all anymore?

Reply:
No, removed both BUILD files.